Cython/Coverage.py

```diff
@@ -218,6 +218,7 @@ def _parse_cfile_lines(self, c_file):
             r'(?:struct|union|enum|class)'
             r'(\s+[^:]+|)\s*:'
         ).match
+        no_cover = re.compile(r'#\s*pragma:\s*no\s+cover\s*').search
```

Comment:
The `\s*` at the end is useless. At least without a `$`. Not sure if I want to add that. Let's relax it for now.
```suggestion
        no_cover = re.compile(r'#\s*pragma:\s*no\s+cover').search
```

Reply:
done